csharp/ql/src/Telemetry/ExternalApi.qll

```diff
@@ -0,0 +1,30 @@
+/** Provides classes and predicates related to handling APIs from external libraries. */
+
+private import ExternalApiSpecific
+
+/**
+ * An external API from either the C# Standard Library or a 3rd party library.
+ */
+class ExternalApi extends ExternalApiSpecific {
```

Comment:
I think the level of sharing here is not enough that we should actually have a shared implementation.

Reply:
Alright. I will remove the sharing!